app/test_main.py

```diff
@@ -1 +1,52 @@
-# write your code here
+from unittest import mock
+
+
+from app.main import can_access_google_page
+
+
+@mock.patch("app.main.valid_google_url")
+@mock.patch("app.main.has_internet_connection")
+def test_access_google_page_with_valid_url_and_time(
+        mocked_url: bool,
+        mocked_internet_connection: bool
+) -> None:
```

Comment:
```suggestion
def test_access_google_page_with_valid_url_and_time(
        mocked_url: bool,
        mocked_internet_connection: bool
       ) -> None:
```
and the same for others